client/client/src/components/AddAnimals.js

```diff
@@ -1,45 +0,0 @@
-import React, { Component } from 'react'
-import axios from 'axios'
-
-const BASE_URL = 'http://localhost:4000'
-
-class AddAnimals extends Component {
-  constructor (props) {
-    super(props)
-    this.state = {
-        name: '',
-        imageUrl:'',
-      };
-    this.handleChange = this.handleChange.bind(this);
-  }
-  
-  handleChange(e) {
-    this.setState({ [e.target.name]: e.target.value})
-  }
```

Comment:
remember you don't need to do the binding in the constructor if you do this syntax: 
```
handleChange = (e) => {

}
```